metricbeat-tests/configurations/redis.yml

```diff
@@ -5,8 +5,14 @@ metricbeat.modules:
   enabled: true
 
   # Redis hosts
-  hosts: ["http://${HOST}:6379"]
+  hosts: ["http://${MONITORED_HOST}:6379"]
 
-output.file:
-  path: "/tmp"
-  filename: "${FILE_NAME}.metrics"
+output.elasticsearch:
+  hosts: ["http://elasticsearch:9200"]
+  username: "elastic"
```

Comment:
Will it be possible to setup those values as variables somehow? Just in case in the future, we would like to use nonephemeral instances but we need to secure them with `vault` or something like that. I don't see this is a blocker but something to think about if it could be required in the future.

Reply:
Yeah, @jsoriano proposed using env vars to completely replace configuration files, which was the initial plan

Reply:
We finally moved to the following approach: specific module setups will go in the YAML file, but the shared ones (elasticsearch output and kibana setup) will be defined as flags for the metricbeat instance